### ISSUE TITLE
Unhandled rejection ReqlRuntimeError on first startup

### ISSUE THREAD

**Issue by Panoplos** [casoetan/react-horizon#3]
I have installed your project per the instruction and upon first run get the following error:

```js
Unhandled rejection ReqlRuntimeError
    at Connection._processResponse (/Users/Panoplos/Documents/Development/react-horizon/node_modules/rethinkdbdash/lib/connection.js:396:15)
    at Socket.<anonymous> (/Users/Panoplos/Documents/Development/react-horizon/node_modules/rethinkdbdash/lib/connection.js:201:14)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:563:20)
```

I put a `console.log(response)` in `_processResponse`, which showed the following:

```js
{ t: 18,
  e: 4100000,
  r: [ 'Table `react_horizon.todos` does not exist.' ],
  b: [ 0 ] }
```

Upon inspecting the db, indeed the only tables that exist are:

```js
[
"hz_collections" ,
"hz_groups" ,
"hz_users_auth" ,
"users"
]
```

Am I missing some step in the setup of the app?

**Comment by casoetan** [comment]
Hi @Panoplos You really should not be having this issue, but I have only tested on a Mac, your milage on a windows system may vary.

But my advice, start rethinkdb before starting the app. 

Let me know if that helps. I'll try on a windows as soon as I can.

Cheers

**Comment by Panoplos** [comment]
I am running on Mac and have RethinkDB running in background. I fixed the issue by manually creating the table. Also, there are several other problems I have run into. Will open another issue.

**Comment by casoetan** [comment]
Ok.
